tests/style_test.py

```diff
@@ -0,0 +1,75 @@
+# pylint: disable=redefined-outer-name
+"""Tests for Pygments styles."""
+
+try:
+    from importlib_metadata import entry_points as _load_entry_points
+except ImportError:
+    from importlib.metadata import entry_points as _load_entry_points
```

Comment:
Why not prefer the stdlib version?

Reply:
Because it would require if-else block instead of try-except. This function is present in all supported versions but only in the latest in supports args we use. It is a conditional dependency meaning that it is only installed when stdlib does not accept any arguments and we can rely on its presence.

Reply:
Resolving this thread to unblock merging.